### PR TITLE
201 kontaktformular überarbeiten

### PR DESCRIPTION
**Pull request by baerlach-git**
dieses Mal nur das reine Feature ohne die Änderungen am Footer, die das Scrolling kaputt gemacht haben

### CODE REVIEW

**Review by olschulz — CHANGES_REQUESTED** [review]
Is prettier working for you? The CI shows a bunch of warnings that should've been auto-formatted by prettier 🤔 Designwise, maybe make the archive drop-down the same width as the other form fields and all the labels a bit bigger? The submit button should probably be the same size and color as the red buttons we have in e.g. the new discover view or the archive view. The end position of the button should probably also be at the end position of the form fields

File: projects/bp-gallery/src/components/views/contact/ContactFormView.scss
Comment (ole1711) [inline projects/bp-gallery/src/components/views/contact/ContactFormView.scss:1]: eventuell die scss mit tailwind komplett umsetzen?

**Review by ole1711 — APPROVED** [review]

File: projects/bp-gallery/src/components/views/contact/ContactFormView.tsx
Comment (olschulz) [inline projects/bp-gallery/src/components/views/contact/ContactFormView.tsx:33]: probably shouldn't still be here right?

**Review by olschulz — APPROVED** [review]
Should be good to go after merging and getting rid of the 'Test' option

File: projects/bp-gallery/src/components/views/contact/ContactFormView.tsx
Comment (olschulz) [inline projects/bp-gallery/src/components/views/contact/ContactFormView.tsx:15]: maybe use asApiPath function from app-helpers.ts?

Comment (MariusDoe) [reply projects/bp-gallery/src/components/views/contact/ContactFormView.tsx:15]: I agree, especially because I think `harz-history` is missing a `.de` here. This would be fixed by using `asApiPath`.

File: projects/bp-strapi/src/api/contact/config/adresses.json
Comment (MariusDoe) [inline projects/bp-strapi/src/api/contact/config/adresses.json:3]: Should `"Test"` still be here?

File: projects/bp-strapi/src/api/contact/controllers/contact.js
Comment (MariusDoe) [inline projects/bp-strapi/src/api/contact/controllers/contact.js:11]: You could use object destructuring here:
```suggestion
    const { sender_name, email: reply_email, subject, message } = body;
```

File: projects/bp-gallery/src/components/views/contact/ContactFormView.tsx
Comment (MariusDoe) [inline projects/bp-gallery/src/components/views/contact/ContactFormView.tsx:64]: I think you want `variant='contained'` here. Then you can use `className='!bg-[#7e241d]'` to set the color instead of the `sx=`. `color: 'white'` will be provided by the `variant`. Although I think we should really use Mui's [Theming](https://mui.com/material-ui/customization/theming/) here to change the primary color and then use `color='primary'`. I have opened #288.

**Review by MariusDoe — CHANGES_REQUESTED** [review]

File: projects/bp-gallery/src/components/views/contact/ContactFormView.tsx
Comment (baerlach-git) [reply projects/bp-gallery/src/components/views/contact/ContactFormView.tsx:64]: Reasonable suggestion, but for now I will leave it as is. Once we have the theme set up, I'll gladly refactor it to use the theme

**Pull request merged**
